app/templates/direct-award/index.html

```diff
@@ -0,0 +1,71 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+{% block page_title %}Saved searches - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": url_for('main.index'),
+          "label": "Digital Marketplace"
+      },
+      {
+          "link": "/buyers",
+          "label": "Account Overview"
+      },
+      {
+          "label": "Saved searches"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="direct-award-saved-searches">
+  <header class="page-heading">
+    <h1>
+      Saved searches
+    </h1>
+    <p class="context">Find, compare and buy cloud services through the G-Cloud framework.</p>
```

Comment:
This content needs to come via the frameworks repo using the content loader. Any content that ties a page to a particular framework should be dynamically loaded. Also as others have said, we shouldn't use the word 'framework' even for testing. Let's just drop `through the G-Cloud framework` altogether. The first half of the sentence should be built as `Find, compare and buy {{ framework_short }}`, where framework_short can be found in dm-frameworks/frameworks/g-cloud-9/messages/descriptions.yml